style.css

```diff
@@ -639,4 +698,143 @@ img:hover {
   .CS-list-items {
     flex-direction: column;
   }
+
+  .popup-txtarea {
+    display: grid;
+    grid-template-columns: calc(60%) calc(40%);
+    grid-template-rows: calc(10%) calc(90%);
+    grid-template-areas:
+      "popuptxt popuplist"
+      "popuptxt popupbtn";
+    gap: 10px;
+    margin: 10px 42px;
+  }
+
+  #proj-primarytxt-id {
+    grid-area: popuptxt;
+  }
+
+  #popuplist {
+    grid-area: popuplist;
+  }
+
+  .style-popupbtn {
+    grid-area: popupbtn;
+  }
+
+  .prj-popup-header > .x-icon {
+    top: 40px;
+    padding-right: 8%;
+  }
+
+  .popup-detailImg {
+    /* max-height: max-content; */
+    height: 40%;
+    width: 90%;
+    align-self: center;
+  }
+
+  .popup-detailCard {
+    flex-direction: column;
+    box-sizing: border-box;
+    margin: 16px auto 50px auto;
+    height: max-content;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    z-index: 3;
+    padding: 16px;
+    max-width: 1156px;
+  }
+
+  #sourcebtn-id,
+  #livebtn-id {
+    box-sizing: content-box;
+  }
+
+  /* Media query for desktop ENDS here */
+}
+
+/* Detail popup section bg */
+
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED:** Kindly make sure you remove all the commented code lines in your code base to make it cleaner and professional. Apply this for the whole page.

Reply:
Oh, I thought that its good to have comments and it was easy to navigate in this way